tests/ci.js

```diff
@@ -18,11 +17,13 @@ const resultSuccess = 0;
 const resultErrored = 1;
 const resultFailureThreshold = 2;
 
-function execElmTest(args) {
+function execElmTest(args, cwd) {
+  // default: current directory
+  cwd = typeof cwd !== 'undefined' ? cwd : '.';
```

Comment:
I think this should work:

```suggestion
function execElmTest(args, cwd = '.') {
```

Reply:
Will this work in node 8?

Reply:
In fact, I first tried it with the default argument syntax (cwd = '.'), but I got test failures - apparently, this syntax was introduced in ES 2015 (tbh, I don't know which version node 8 is based on). So I guess we're stuck with the more cumbersome approach (cwd = typeof ...) for the time being.

Reply:
Dropping node 8 support is on our short term road map. We can fix this once we have done that.